configuration.go

```diff
@@ -173,6 +175,43 @@ type TLS struct {
 // Certificates defines traefik certificates type
 type Certificates []Certificate
 
+//CreateTLSConfig creates a TLS config from Certificate structures
```

Comment:
Nope, just explain in the struct comment that files can be either path or content ;)